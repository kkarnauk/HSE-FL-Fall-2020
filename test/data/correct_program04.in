ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff :- fffffffffffffffffffffffffffffffffffffffffffff hgfshfdg fshgdsjhgsjd sdhgsajd
sdgh jhgdhsgajd gsjdhg ajshdg ajshgdj asgdja gjdgaj.
