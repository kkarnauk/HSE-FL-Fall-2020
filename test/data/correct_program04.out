|relation
|	|head
|	|	|identifier = 'ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff'
|	|body
|	|	|atom
|	|	|	|identifier = 'fffffffffffffffffffffffffffffffffffffffffffff'
|	|	|	|identifier = 'hgfshfdg'
|	|	|	|identifier = 'fshgdsjhgsjd'
|	|	|	|identifier = 'sdhgsajd'
|	|	|	|identifier = 'sdgh'
|	|	|	|identifier = 'jhgdhsgajd'
|	|	|	|identifier = 'gsjdhg'
|	|	|	|identifier = 'ajshdg'
|	|	|	|identifier = 'ajshgdj'
|	|	|	|identifier = 'asgdja'
|	|	|	|identifier = 'gjdgaj'